run_robot_test.sh

```diff
@@ -55,6 +63,9 @@ handle_inputs() {
             v) # Override/Add global variables specified in the test variables file
                 TEST_VARIABLES="${TEST_VARIABLES} --variable $OPTARG"
             ;;
+            x) # Override variable file
+                override_url_variables
+            ;;
             *)
```

Comment:
Is the '*' necessary here?

Reply:
Yes @ikanias This is for default case. Please find the details  - https://www.tutorialspoint.com/unix/case-esac-statement.htm

Reply:
Thanks for the answer!